resources/styles/book-content/ap-bio/ap-connection.less

```diff
@@ -0,0 +1,71 @@
+.ap-connection{
+   h1 + p:first-of-type{
```

Comment:
Can you make sure the indentation is consistent? 

- 1 space between selectors and `{`
- 2 spaces for indent